src/comm/Tcp.cc

```diff
@@ -15,6 +15,9 @@
 #if HAVE_NETINET_TCP_H
 #include <netinet/tcp.h>
 #endif
+#if HAVE_NETINET_IN_H
+#include <netinet/in.h>
+#endif
```

Comment:
This change also does not appear to be related to the rest of this patch. If this is a second issue being fixed please submit it separately.

Reply:
ditto.